lib/rubocop/cop/style/yoda_expression.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Forbids Yoda expressions,
```

Comment:
I'd suggest being more explicit about the exact scope of the cop, as it covers a relatively small number of expressions. Perhaps we should also mention it complements `YogaConditions`.